maintainers/maintainer-list.nix

```diff
@@ -5034,6 +5034,12 @@
     name = "Edvin Källström";
     githubId = 84442052;
   };
+  ekimber  = {
+    email = "ekimber@protonmail.com";
+    github = "ekimber";
+    name = "Edward Kimber";
+    githubId = 99987;
+  };
```

Comment:
This should be committed by itself and appear as the first commit in this pr.

The format for the commit should then be `maintainers: add ekimber`